src/common/transformations/tests/op_conversions/convert_weight_compressed_conv1x1_to_matmul_test.cpp

```diff
@@ -27,8 +27,8 @@
 #include "openvino/opsets/opset7_decl.hpp"
 #include "transformations/rt_info/decompression.hpp"
 
+using namespace ov;
```

Comment:
The test coverage seems limited. Currently there are only two test cases: one positive (1x1 conv transforms to matmul) and one negative (3x3 conv does not transform). Are there plans to add tests for:
asymmetric quantization path (with zero-point subtraction)
bias addition path
Parameter weights instead of Constant
Reshape on output instead of Transpose
Convert on output